routes/votings/list-of-votings-api.js

```diff
@@ -5,9 +5,10 @@ const express = require('express');
 const db = require("./list-of-votings-f");
 
 const REACT_JS_ROOT = process.env.REACT_JS_ROOT_URL
+const REACT_ALT_ROOT = process.env.REACT_ALT_ROOT_URL
 const router = express.Router();
 router.use(cors({
-    origin: `${REACT_JS_ROOT}`
+  origin: [`${REACT_JS_ROOT}`, `${REACT_ALT_ROOT}`]
```

Comment:
I would use only one env var for CORS whose value would be a list of allowed origins (or any other value the [`origin` option of the cors middleware](https://expressjs.com/en/resources/middleware/cors.html#:~:text=Configuration%20Options-,origin,-%3A%20Configures%20the%20Access)).

I would also rename it to something like `ALLOWED_ORIGINS` because this application should not need to know anything about technical details of the origin, like whether it's using React.

```js
router.use(cors({
  origin: process.env.ALLOWED_ORIGINS,
}))
```

Reply:
done